boost-common/src/main/java/io/openliberty/boost/common/boosters/CDIBoosterConfig.java

```diff
@@ -28,11 +29,8 @@
 
     public CDIBoosterConfig(Map<String, String> dependencies, BoostLoggerI logger) throws BoostException {
         String version = dependencies.get(getCoordindates(this.getClass()));
-        
-        // if it is the 1.0 version = EE7 feature level
-        if (version.equals(MP_20_VERSION)) {
-            libertyFeature = CDI_20;
-        }
+        Properties boostConfigProperties = BoostProperties.getConfiguredBoostProperties(logger);
```

Comment:
What is the idea in getting the `boostConfigProperties` in every booster? I don't see any of them being used right now.

Reply:
ok, i re-worked all of this onto the latest master, i think all will be resolved, but you can re-look at it if need be.